### PR TITLE
Feature/refactor event system

### PR DESCRIPTION
**Pull request by Krischan-Klug**

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+export const petEvents = [
+  {
+    id: 1,
+    eventName: "Your pet is sick",
+    description: `your pet is sick. You need to pay 100 medical costs. and your pet lost 20 engergy.`,
```
[the remark this answers is not in the document]
Comment (CJAkkuec) [reply lib/events.js:5]: I've noticed several typos/etc. This is just a representative correction. Perhaps you should have a member of your team proof-read the events. :) Usually VSC has a spellchecker, iirc.

File: lib/events.js [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    },
+  },
+];
+
+export const userEvents = [
+  {
+    id: 1,
+    eventName: "Wrong investment",
+    description:
+      "You bought the wrong cryptocurrency. You lose 200 of your money.",
```
Comment (CJAkkuec) [inline lib/events.js:165]: They're all bad. :D

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
Looking very good overall, but the events may need some spellchecking or grammatical adjustments.

File: lib/events.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+      hunger: 0,
+      energy: -20,
+      money: -100,
+    },
+  },
+
+  {
+    id: 2,
+    eventName: "Birthday Party",
+    description: `It's your pet birthday and you're throwing a big party with all of his pet friends. Pay 50 pet coins.`,
```
Comment (jana-scheidemann) [inline lib/events.js:17]:
```suggestion
    description: `It's your pet's birthday and you're throwing a big party with all of its pet friends. Pay 50 pet coins.`,
```

File: lib/events.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+      happiness: -100,
+      hunger: -100,
+      energy: -100,
+      money: 0,
+    },
+  },
+  {
+    id: 6,
+    eventName: "Archaeological find",
+    description: `your pet digs up an archaeological find. You sell it for 1200 pet coins.`,
```
Comment (jana-scheidemann) [inline lib/events.js:61]:
```suggestion
    description: `Your pet digs up an archaeological find. You sell it for 1200 pet coins.`,
```

File: lib/events.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+      happiness: 0,
+      hunger: 0,
+      energy: 0,
+      money: 1200,
+    },
+  },
+  {
+    id: 7,
+    eventName: "Mr/s barking",
+    description: `your pet stays up all night barking at the neighbor's cat. Energy decreases by 30.`,
```
Comment (jana-scheidemann) [inline lib/events.js:72]:
```suggestion
    description: `Your pet stays up all night barking at the neighbors cat. Energy decreases by 30.`,
```

File: lib/events.js [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+      happiness: 0,
+      hunger: 0,
+      energy: -30,
+      money: 0,
+    },
+  },
+
+  {
+    id: 8,
+    eventName: "baking hour",
```
Comment (jana-scheidemann) [inline lib/events.js:83]: If all your events start with a capital letter, this should, too.
```suggestion
    eventName: "Baking hour",
```

File: lib/events.js [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+    description: `your pet smells freshly baked cookies and begs for a treat. Hunger increases by 15.`,
```
Comment (jana-scheidemann) [inline lib/events.js:84]:
```suggestion
    description: `Your pet smells freshly baked cookies and begs for a treat. Hunger increases by 15.`,
```

File: lib/events.js [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+      happiness: 0,
+      hunger: -15,
+      energy: 0,
+      money: 0,
+    },
+  },
+  {
+    id: 9,
+    eventName: "Lost toy",
+    description: `your pet favorite toy gets lost. Happiness decreases by 40.`,
```
Comment (jana-scheidemann) [inline lib/events.js:95]: Please check the rest of the events yourself! 🧑🏻‍🏫

**Review by jana-scheidemann — COMMENTED** [review]
Hey Krischan, I checked only what CJ mentioned. Unfortunately you did not check good enough because I still find a lot of typos. After doing half of it for you I decided to let you do the rest of it 😉 HOWEVER, I had a lot of fun reading the events 😁 Your app is so cool 🎊 and I am impressed by what you have done so far.

File: lib/events.js (hunk 5 above)
Comment (CJAkkuec) [reply lib/events.js:72]: Correcting your correction with `neighbor's` was fine, unless it's plural, then it'd be `neighbors'`. 😂

File: lib/events.js (hunk 8 above)
Comment (Krischan-Klug) [reply lib/events.js:95]: Sry my english not the best

**Pull request merged**